src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -476,15 +479,19 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
     device: Device,
     fromAcquire: boolean
   ): Promise<DevicePermission> {
+    this.videoInputDeviceCount += 1;
```

Comment:
Should this only increase if kind === 'video'

Reply:
A good catch - This counter might be used for audio later. I'll rename it to `this.inputDeviceCount`.

Reply:
Yes, I was about to ask whether we have the same issue for ChooseAudio<Input/Output>Device?

Reply:
- Both APIs have the same issue of not releasing a previous device if the API is getting multiple times.

- But calling `chooseAudioInputDevice(null)` creates an empty audio device, while calling `chooseVideoInputDevice(null)` unselects devices. I haven't looked into details for the audio flow.